static: work around go get 404 failure

kortschak opened the pull request:
This now includes quotes on the meta tag name to work around the incorrect
rendering by hugo(?).

Again, I'm going to merge to see if this fixes the problem.

Updates #9,

sbinet commented:
@btracey did I do everything correctly?

I tried looking at the end result with `hugo serve` but I got a blank page...

btracey commented:
The command is `hugo serve --watch`

btracey commented:
Let me know if I should download to check.

btracey commented on content/post/intro-to-stats-with-gonum.md:
```diff
@@ -0,0 +1,203 @@
+---
+title: "Introduction to Statistics With Gonum"
+date: 2017-10-04T11:17:11+02:00
+Categories: ["go", "gonum", "statistics"]
```
I'm not sure how we want to do this, but I think at least `stat` should be one of the categories to explicitly tag it to the package name. I'm not sure what that means about the `statistics` tag. Do you have opinions @kortschak ?

btracey commented on content/post/intro-to-stats-with-gonum.md:
```diff
@@ -0,0 +1,203 @@
+Categories: ["go", "gonum", "statistics"]
+---
+
+_The following was cross-posted from [sbinet.github.io/posts/2017-10-04-intro-to-stats-with-gonum](https://sbinet.github.io/posts/2017-10-04-intro-to-stats-with-gonum/)_
```
"cross-posted with minor changes" (because I'm going to make some suggestions)

sbinet replied:
done.

sbinet commented:
`hugo serve` also works.
I was probably getting a blank page because I had no wireless ?
not sure.

in any case, it looks fine here :)

btracey commented on content/post/intro-to-stats-with-gonum.md:
```diff
@@ -0,0 +1,203 @@
+
+_The following was cross-posted from [sbinet.github.io/posts/2017-10-04-intro-to-stats-with-gonum](https://sbinet.github.io/posts/2017-10-04-intro-to-stats-with-gonum/)_
+
+Starting a bit of a new series (hopefully with more posts than with the interpreter ones) about using [Gonum](https://gonum.org) to apply statistics.
```
It would be nice if this post had a gonum flair in the intro. "This is the first of a series of short posts providing an introduction and code examples for using the gonum packages. This first post focuses on computing basic statistics using the `stat` package"

sbinet replied:
done.

btracey commented on content/post/intro-to-stats-with-gonum.md:
```diff
@@ -0,0 +1,203 @@
+
+This first post is really just a copy-paste of this one:
+
+ https://mubaris.com/2017-09-09/introduction-to-statistics-using-numpy
```
I think it's unfair to you to say it's "just a copy-paste". I would say something like "is based on [url]"

sbinet replied:
done.

btracey commented on content/post/intro-to-stats-with-gonum.md:
```diff
@@ -0,0 +1,203 @@
+
+[Gonum](https://gonum.org) is _"a set of packages designed to make writing numeric and scientific algorithms productive, performant and scalable."_
+
+Before being able to use `Gonum`, we need to install [Go](https://golang.org).
```
Maybe for the Gonum blog this could just link to the "intro to gonum" rather than repeat installation instructions? That way if things change we only need to fix once.

sbinet replied:
done.

btracey commented on content/post/intro-to-stats-with-gonum.md:
```diff
@@ -0,0 +1,203 @@
+
+	fmt.Printf("data: %v\n", xs)
+
+	sort.Float64s(xs)
```
I would do

Mean
Variance
StdDev
Sort
Quantile

The first couple don't need the data sorted, but Quantile does. That ordering would make it more clear.

sbinet replied:
done.

btracey commented on content/post/intro-to-stats-with-gonum.md:
````diff
@@ -0,0 +1,203 @@
+```
+
+This is because `numpy.var` uses `len(xs)` as the divisor while `gonum/stats`
+uses the unbiased sample variance (_ie:_ the divisor is `len(xs)-1`):
````
Maybe note that `Moment` can be used for the uncorrected estimator?

sbinet replied:
done.

btracey commented on content/post/intro-to-stats-with-gonum.md:
````diff
@@ -0,0 +1,203 @@
+std-dev=  55194.44110508921
+```
+
+The data file can be obtained from here: [salary.txt](https://sbinet.github.io/code/2017-10-04/salary.txt)
````
Do we want to vendor these files on the Gonum site?

sbinet replied:
it's up to you.
in case you do, just let me know whether there is a recommended place to put them.

btracey replied:
Maybe in static? Do you know @kortschak ?

sbinet replied:
in my blog, I put them under `/static/code/<post-date>/...`

sbinet replied:
(and I put `.go`, `.cxx`, `.png`, ... files)

btracey replied:
That sounds like a great location.

btracey commented on content/post/intro-to-stats-with-gonum.md:
````diff
@@ -27,11 +28,7 @@ Once that has been done, installing `Gonum` and all its dependencies can be done
-```sh
-$> go get -u gonum.org/v1/gonum/...
-```
+Please refer to this [post](/post/introtogonum) for more details on Gonum.
````
Maybe delete the "# Go & Gonum" section, and instead: 

```
 but using [Go](https://golang.org) and [Gonum](https://gonum.org) instead of Python and `numpy`.

[Gonum](https://gonum.org) is _"a set of packages designed to make writing numeric and scientific algorithms productive, performant and scalable."_
Please refer to this [post](/post/introtogonum) for more details on Gonum.
```

sbinet replied:
done

vladimir-ch commented on content/post/intro-to-stats-with-gonum.md:
```diff
@@ -0,0 +1,190 @@
+
+but using [Go](https://golang.org) and [Gonum](https://gonum.org) instead of Python and `numpy`.
+
+[Gonum](https://gonum.org) is _"a set of packages designed to make writing numeric and scientific algorithms productive, performant and scalable."_
```
I have brought this up before: I have never seen "numeric" used in our context, it is always "numerical".

sbinet replied:
done.

sbinet commented:
tentatively vendored files under `/static/code/intro-to-stats-with-gonum`

btracey commented on content/post/intro-to-stats-with-gonum.md:
````diff
@@ -0,0 +1,190 @@
+
+[embedmd]:# (../../static/code/intro-to-stats-with-gonum/stats-salary.go go)
+```go
+// file: stats-salary.go
````
I noticed this shows up in the markdown. Not sure if that's intentional

btracey commented on content/post/intro-to-stats-with-gonum.md:
````diff
@@ -0,0 +1,190 @@
+std-dev=  55194.44110508921
+```
+
+The data file can be obtained from here: [salary.txt](/code/intro-to-stats-with-gonum/salary.txt)
````
I haven't seen markdown with relative links before. In the preview, this links to `github.com/sbinet/stuff`. Will it link to `gonum.org` when merged?

sbinet replied:
yep.

btracey commented on content/post/intro-to-stats-with-gonum.md:
````diff
@@ -0,0 +1,190 @@
+
+[embedmd]:# (../../static/code/intro-to-stats-with-gonum/stats.go go)
+```go
+// file: stats.go
````
This shows up in the markdown, not sure if intentional.

sbinet replied:
what exactly are you talking about ?
`[embedmd]:...` is a nice little pre-processor from Francesc: [embedmd](https://github.com/campoy/embedmd)
you give it a path to a file and it will copy it right under the `[embedmd]` markup.

and in that `stats.go` file, I do have as a first line `// file: stats.go`.
I felt it was easier to understand as I was saying later on `go run ./stats.go`.

btracey replied:
Great, just wanted to check that was intentional.

sbinet commented:
Let me squash this a bit.

sbinet commented:
done.

kortschak committed:
static: work around go get 404 failure

This now includes quotes on the meta tag name to work around the incorrect
rendering by hugo(?).
